kategory-effects-rx2/src/main/kotlin/kategory/effects/instances/FlowableKWInstances.kt

```diff
@@ -0,0 +1,129 @@
+package kategory
+
+import io.reactivex.BackpressureStrategy
+
+object FlowableKWMonadInstanceImplicits {
```

Comment:
These explicit objects are now automatically generated, please merge master and see how other instances have their boilerplate for these objects and factory methods that traditionally have been added to companions automatically generated since #288

Reply:
Nevermind I see that this is similar to Observable with 3 possible instances for Monad

Reply:
There's a default one. Observable and Flowable having 3 is my mistake. I've spoken with @anstaendig to fix it in this branch before merging.